src/init.js

```diff
@@ -0,0 +1,62 @@
+const path = require("path");
+const { readFileSync } = require("fs");
+const db = require('./db')
+const schemaValidator = require("./utils/schemaValidator")
+
+const readFileContent = () => {
+  try {
+    const absolutePath = path.join(process.env.PWD, 'init.json');
+    const jsonString = readFileSync(absolutePath, 'utf-8');
+    return JSON.parse(jsonString);
+  } catch (e) {
+    console.error(e);
+  }
+}
+
+module.exports = async () => {
+  console.info('Starting initialization...');
```

Comment:
Should we check the file to exist first? I mean, if there is no `init.json`, then there should be no logs related to initialization or some `Init file not found, skipping initialization`